packages/discovery-react-components/src/components/SearchResults/utils/getDocumentTitle.ts

```diff
@@ -0,0 +1,18 @@
+import DiscoveryV2 from 'ibm-watson/discovery/v2';
```

Comment:
I think we should have this file at the top-level `utils` folder, not within `SearchResults/`

Reply:
wouldn't it best to leave it here since it only concern the `SearchResults` and unlikely to be used anywhere else

Reply:
Well, in this case we're looking to export it as part of our public API, and not as a member of `SearchResults`. For that reason, I think our code layout should match our API.

Reply:
Fair enough 👍